src/test/java/teammates/test/cases/browsertests/FeedbackRubricQuestionUiTest.java

```diff
@@ -68,36 +68,35 @@ private void testInstructorResultsPage() throws Exception {
         InstructorFeedbackResultsPage instructorResultsPage =
                 loginToInstructorFeedbackResultsPageWithViewType("teammates.test.instructor", "openSession2",
                                                                  false, "question");
-        instructorResultsPage.waitForPanelsToExpand();
-
+        clickAjaxLoadedPanelAndWaitForExpansion(instructorResultsPage, "panelHeading-1", "ajax_auto");
```

Comment:
I don't think there's a need to test this multiple times, if we test it once and it works, great - especially since an ajax call extends our test time